keyboards/4pplet/bootleg/README.md

```diff
@@ -0,0 +1,12 @@
+# bootleg
```

Comment:
Please rename this file to lowercase.

Reply:
Thanks for the feedback and changes! Filename is changed to lower case.